best-practices/README.md

```diff
@@ -101,6 +101,10 @@ Rails
   environment variables are detected on deploy.
 * [Use blocks][date-block] when declaring date and time attributes in FactoryGirl factories.
 * Use `touch: true` when declaring `belongs_to` relationships.
+* Name service classes after the action they perform
```

Comment:
@calleerlandsson I like that rule—I could see it encouraging more generality from objects `lib`. 

I also think there are several ways for a class to be "a part of the application's business logic/domain model." The rule of thumb I've been using is to distinguish between encapsulating and coordinating classes: i.e., between those primarily meant to encapsulate business logic (models) and those primarily meant to coordinate the behavior of models (services) (in order to unburden controllers so they can stay focused on the request/response cycle). 

On way that rule of thumb manifests is that objects of the latter category tend to have more external dependencies, which is one reason I like to group them in a separate directory (`app/services`) -- the difference between files in `models` and `services` ends up being a visual cue reminding you to minimize dependencies when working with model logic.